src/Forms/Form.php

```diff
@@ -402,6 +411,21 @@ protected function getFormPlaceholders(array $fields)
         return $attributes;
     }
 
+    /**
+     * Set the validator return value on fail to true.
+     * Pass false as an arugment to put it back to false.
+     *
+     * @param bool $value
+     *
+     * @return $this
+     */
+    public function setValidatorReturnValueOnFail($value = true)
+    {
+        $this->validatorReturnValueOnFail = $value;
+
+        return $this;
+    }
+
```

Comment:
```suggestion
```

You can delete this method. Make sure to add the `flushOnFail` option key to Form `$allowedOptions` property and add `'flushOnFail' => true` to the `$defaultOptions` property.